lib/metadataTypes/MetadataType.js

```diff
@@ -351,6 +352,8 @@ class MetadataType {
                         metadata[metadataKey],
                         deployDir
                     );
+
+                    console.log('deployableMetadata', deployableMetadata);
```

Comment:
please revert entire commit of this file (right click in sourcetree, revert commit)